### PR TITLE
mixed poisson: add from firedrake import * 

### PR DESCRIPTION
**Pull request by w3168**
mixed poisson had lost 'from firedrake import *

On the first commit I think all the jupyter notebook runtime stuff got added, 
which hopefully was removed on the second commit. Happy to tidy it up more!

### CODE REVIEW

File: 05-mixed-poisson.ipynb [hunk 1]
```diff
@@ -1,519 +1,608 @@
...
+            "\u001b[0;31mNameError\u001b[0m: name 'FiniteElement' is not defined"
```
Comment (joewallwork) [inline 05-mixed-poisson.ipynb]: Hm it looks like something is broken?

**Review by joewallwork — CHANGES_REQUESTED** [review]
Please could you check what is going on with that error? Also, please could you clear all output so that the attendees run stuff for themselves? Thanks

File: 05-mixed-poisson.ipynb (hunk 1 above)
Comment (w3168) [reply 05-mixed-poisson.ipynb]: Ah sorry Joe that would also be fixed by 'from firedrake import *' 

I'll tidy up the outputs

**Comment by w3168** [comment]
Hi Joe has this removed the outputs? 

I've hopefully gone back to the original notebook and then added the 'from firedrake import *' without running.

**Comment by joewallwork** [comment]
Yes, seems fine. I assume the cells all run okay?

**Comment by w3168** [comment]
> Yes, seems fine. I assume the cells all run okay?

Yep just tested now pointing colab to w3168/firedrake-tutorial and the mixed poisson example works fine.

**Pull request merged**